anomalous-value/bot-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "developerAbbreviation": "AE",
+  "protocolName": "Aave",
+  "protocolAbbreviation": "AAVE",
+  "contract": {
```

Comment:
Mega Nit: In other `bot-config.json` files, this is plural, `contracts` as opposed to `contract`

Reply:
This was intentional, as the current implementation of `anomalous-value` only works with one specific `contract`. This is outside the scope of this ticket, but it may be worth investigating how to generalize this bot so that it could handle multiple `contracts`.